### PR TITLE
fix: pagination page size in message list

### PR DESCRIPTION
**Pull request by maze-runnar**
related https://github.com/fossasia/open-event-frontend/issues/3693

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/routes/admin/messages/list.js [hunk 1]
```diff
@@ -7,9 +7,9 @@ export default class extends Route.extend(EmberTableRouteMixin) {
     const searchField = 'recipient';
     const filterOptions = this.applySearchFilters([], params, searchField);
     let queryString = {
-      filter        : filterOptions,
-      'page[size]'  : params.per_page || 100,
-      'page[number' : params.page || 1
+      filter         : filterOptions,
+      'page[size]'   : params.per_page || 100,
+      'page[number]' : params.page || 1
```
Comment (iamareebjamal) [inline app/routes/admin/messages/list.js:12]: What does this change?

Comment (maze-runnar) [reply app/routes/admin/messages/list.js:12]: I was looking into https://github.com/fossasia/open-event-frontend/issues/3693 then I found the wrong syntax `page[number`

Comment (maze-runnar) [reply app/routes/admin/messages/list.js:12]: the problem with #3693 is due to wrong passing of querystring. I have some doubts regarding that to discuss in technical meeting.

File: app/routes/admin/messages/list.js [hunk 2]
```diff
@@ -7,9 +7,9 @@ export default class extends Route.extend(EmberTableRouteMixin) {
...
     };
     queryString = this.applySortFilters(queryString, params);
     return this.asArray(this.modelFor('admin.messages', queryString));
```
Comment (maze-runnar) [inline app/routes/admin/messages/list.js:15]: Actually the queryString should be in 'admin.messeges' with page[size] and page[number]. But how to pass it there?

**Comment by maze-runnar** [comment]
I am getting this error after moving mesages/list to mesages and deleting it.
Screenshot from 2021-01-08 09-24-13

**Comment by maze-runnar** [comment]
the above is fixed, but sorting and searching is giving error.

**Comment by maze-runnar** [comment]
`{"errors": [{"status": 400, "source": {"parameter": "filters"}, "title": "Invalid filters querystring parameter.", "detail": "MessageSettings has no attribute recipient"}], "jsonapi": {"version": "1.0"}}`

**Comment by iamareebjamal** [comment]
Push the changes, or else how will I debug?

**Comment by iamareebjamal** [comment]
See Message Setting model in server and remove sortable from all fields which are not in model, like recipient

**Comment by maze-runnar** [comment]
I have removed recipient and now ony sent_at is sortable. And there is no other field that can be make searchable. So I hid the searchbox.

**Comment by iamareebjamal** [comment]
Is the save button necessary?

**Comment by maze-runnar** [comment]
> Is the save button necessary?

I think no, it's not working right now or it shouldn't be there.

**Comment by iamareebjamal** [comment]
It should not be removed if it was saving the settings, please restore but put it in a proper place

**Comment by maze-runnar** [comment]
> It should not be removed if it was saving the settings, please restore but put it in a proper place

but it's not saving settings actually. they are kind of fixed. On server side the setting is implemented as  

```
def create_admin_message_settings():
    default_mails = [
        "Next Event",
        "Session Schedule Change",
        "User email",
        "Invitation For Papers",
        "After Event",
        "Ticket(s) Purchased",
        "Session State Change",
        "Event Published",
        "Event Export Failed",
        "Event Exported",
        "Event Role Invitation",
        "New Session Proposal",
    ]
    for mail in MAILS:
        if mail in default_mails:
            get_or_create(
                MessageSettings,
                action=mail,
                mail_status=True,
                notification_status=True,
                user_control_status=True,
            )
        else:
            get_or_create(
                MessageSettings,
                action=mail,
                mail_status=False,
                notification_status=False,
                user_control_status=False,
            )
```

**Comment by maze-runnar** [comment]
so on refreshing page the settings are set to previous(the above values), no matter they are saved whatever

**Comment by iamareebjamal** [comment]
But notification and mail status are editable

**Comment by maze-runnar** [comment]
reverted

**Comment by iamareebjamal** [comment]
Move save to right and add padding top

**Comment by maze-runnar** [comment]
is it ok?
Screenshot from 2021-01-08 17-05-59

**Pull request merged**